controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -151,10 +151,20 @@ def test_create_volume_with_compressed_space_efficiency_success(self):
         self.svc.client.svctask.mkvolume.assert_called_with(name="test_volume", unit="b", size=1024, pool="pool_name",
                                                             compressed=True)
 
-    def test_create_volume_with_deduplicated_space_efficiency_success(self):
+    def test_create_volume_with_deduplicated_thin_space_efficiency_success(self):
+        self._test_create_volume_success(config.SPACE_EFFICIENCY_DEDUPLICATED_THIN)
+        self.svc.client.svctask.mkvolume.assert_called_with(name="test_volume", unit="b", size=1024, pool="pool_name",
+                                                            deduplicated=True, thin=True)
+
+    def test_create_volume_with_deduplicated_compressed_space_efficiency_success(self):
+        self._test_create_volume_success(config.SPACE_EFFICIENCY_DEDUPLICATED_COMPRESSED)
+        self.svc.client.svctask.mkvolume.assert_called_with(name="test_volume", unit="b", size=1024, pool="pool_name",
+                                                            deduplicated=True, compressed=True)
+
+    def test_create_volume_with_deduplicated_backward_compatibility_space_efficiency_success(self):
         self._test_create_volume_success(config.SPACE_EFFICIENCY_DEDUPLICATED)
         self.svc.client.svctask.mkvolume.assert_called_with(name="test_volume", unit="b", size=1024, pool="pool_name",
-                                                            compressed=True, deduplicated=True)
+                                                            deduplicated=True, compressed=True)
```

Comment:
why this change?

Reply:
I was just rewriting all the dedup,  I didn't remember what was the old test order..

Reply:
done